galter_subjects_utils/contrib/lcsh/cli.py

```diff
@@ -122,26 +128,34 @@ def lcsh_file(**parameters):
 
 @lcsh.command("deprecated")
 @click.argument(
-    "downloads-dir",
-    type=click.Path(path_type=Path, exists=True, file_okay=False),
+    "subjects-file",
+    type=click.Path(path_type=Path, exists=True, dir_okay=False),
 )
-@click.option("--since", "-s", default=None, help="Filter for YYYY-MM-DD and later.")  # noqa
+@click.option(
+    "--since", "-s", default=None, help="Filter for YYYY-MM-DD and later."
+)
+@click.option("--output-file", "-o", type=click.Path(path_type=Path))
 def lcsh_deprecated(**parameters):
     """Generate CSV file of raw deprecated LCSH topics.
 
     This file is then parsed by a metadata expert and potentially edited
     for future use in lcsh_deltas.
     """
-    downloads_dir = parameters["downloads_dir"].expanduser()
+    fp_of_subjects = parameters["subjects_file"].expanduser()
```

Comment:
expanduser() isn't really necessary in practice and might be removed in the future.